src/components/LoginPage.tsx

```diff
@@ -0,0 +1,71 @@
+import React, {FormEvent, useState} from 'react';
+import {useForm} from 'react-hook-form';
+import Logo from "../assets/logo.png";
+
+interface LoginCredentials {
+    email: string,
+    password: string,
+}
+export const LoginPage: React.FC = () => {
+    const [email, setEmail] = useState<string>('')
+    const [password, setPassword] = useState<string>('')
+
+    const {register} = useForm<LoginCredentials>();
+
+    const handleSubmit = (e: FormEvent) => {
+        e.preventDefault()
+        console.log(email, password)
+
+    }
+
+    const flexCenter = "flex flex-col items-center justify-center";
```

Comment:
Z tego co czytałem w dokumentacji tailwinda najlepiej jest wszystko umieszczać w className, wiem, że przez to piszemy długie linijki ale dążymy żeby wszystkie klasy były w jednym miejscu dla konwencji i utrzymania kodu.